framework/utils/CPropertyValue.php

```diff
@@ -113,8 +113,14 @@ public static function ensureArray($value)
 			$len = strlen($value);
 			if ($len >= 2 && $value[0] == '(' && $value[$len-1] == ')')
 			{
-				eval('$array=array'.$value.';');
-				return $array;
+				try
+				{
+					return @eval('return array' . $value . ';');
```

Comment:
That changes behavior to be silent if PHP version is lower than 7. I don't think we should change it.